release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -138,6 +138,22 @@ A new `--node-selector` argument provides a way to identify which nodes you are
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
 
+[id="ocp-4-10-hardware"]
+=== Hardware
+
+[id="ocp-4-10-redfish-events"]
+==== Low-latency Redfish hardware event delivery (Technology Preview)
+{product-title} now provides a hardware event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware events such as hardware changes and failures.
+
+The hardware event proxy supports a publish-subscribe service through which hardware events detected by Redfish can be consumed by relevant applications. The proxy must be running on hardware that supports Redfish v1.8 and higher.
```

Comment:
Can this language be simplified to: ...publish-subscribe service that allows relevant applications to consume hardware events detected by Redfish.